src/components/map/MapComponet.jsx

```diff
@@ -0,0 +1,29 @@
+import { createRef, useEffect, useRef } from "react";
+import 'ol/ol.css'
+import { useMap } from "../../contexts/mapContext/mapContext";
+import DisplayLayers from "../layers/displayLayers";
+import DrawInteraction from "../layers/drawing/drawInteraction";
+import { Box } from "@mui/material";
+
+
+
+const MapComponent = () => {
+    const mapRef = useRef(useMap());
+    const ref = createRef()
+
+    useEffect(() => {
+        mapRef.current.setTarget(ref.current);
+    }, [])
+
+
+    
+    return (
+        <Box>
+          <div ref={ref} style={{ width: '100vw', height: '400px'}}/>

```

Comment:
Given that you're utilizing MUI, you might want to consider substituting the vanilla _div_ with a _Box_ or a _Container_ component.